js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+        //             ...
+        //          ],
+        //         "code":"200",
+        //
+        data = this.safeValue (response, 'result', []);
+        const result = [];
+        for (let i = 0; i < data.length; i++) {
+            const market = data[i];
+            const id = this.safeString (market, 'name');
+            const baseId = this.safeString (market, 'stock');
+            const quoteId = this.safeString (market, 'money');
+            const base = this.safeCurrencyCode (baseId);
+            const quote = this.safeCurrencyCode (quoteId);
+            const symbol = base + '/' + quote;
+            const priceScale = this.safeInteger (market, 'moneyPrec');
+            const quantityScale = this.safeInteger (market, 'stockPrec');
+            const pricePrecision = priceScale;
+            const quantityPrecision = quantityScale;
+            const minAmount = this.safeNumber (market, 'minAmount');
+            const type = 'spot';
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'type': type,
+                'spot': true,
+                'derivative': undefined,
+                'contract': undefined,
+                'linear': undefined,
+                'inverse': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'settle': undefined,
+                'settleId': undefined,
+                'margin': undefined,
+                'active': undefined,
+                'taker': undefined,
+                'maker': undefined,
+                'precision': {
+                    'price': pricePrecision,
+                    'amount': quantityPrecision,
+                },
+                'limits': {
+                    'leverage': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': minAmount,
+                        'max': undefined,
+                    },
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': this.safeNumber (market, 'feePrec'),     // <<<TODO>>> I am not yet sure this is correct.
+                        'max': undefined,
+                    },
+                },
+                'info': market,
+            });
+        }
+        return result;
+    }
+
+    async fetchTicker (symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchTicker requires a symbol argument');
+        }
```

Comment:
Let's fix the signature of this method, the `symbol` is a required argument with all `fetchTicker` implementations everywhere, so we don't need this `if (symbol === undefined)` conditional in `fetchTicker`:

```JavaScript
async fetchTicker (symbol, params = {}) {
   ...
```